sdk/servicebus/azure-servicebus/stress/scripts/logger.py

```diff
@@ -24,20 +24,25 @@ def get_base_logger(log_filename, logger_name, level=logging.ERROR, print_consol
             console_handler.setFormatter(formatter)
             logger.addHandler(console_handler)
     else:
-        # rotated hourly if small file, o/w rotated bi-hourly
-        if level == logging.DEBUG or level == logging.INFO:
-            time = 30
+        if rotating_logs:
+            # rotated hourly if small file, o/w rotated bi-hourly
+            if level == logging.DEBUG or level == logging.INFO:
+                time = 30
+            else:
+                time = 60
+            file_handler = TimedRotatingFileHandler(log_filename, when='M', interval=time, utc=True)
+            if not logger.handlers:
+                file_handler.setFormatter(formatter)
+                logger.addHandler(file_handler)
         else:
-            time = 60
-        file_handler = TimedRotatingFileHandler(log_filename, when="M", interval=time, utc=True)
-        if not logger.handlers:
+            file_handler = logging.FileHandler(log_filename)
```

Comment:
do you want any conditionals here if it's debug or not?

Reply:
I don't think so? Since we'll want to log errors at least, so the logger should always be logging.

Reply:
On second thought, good point! Adding conditionals on debug makes sense so that we're not unnecessarily rotating logs when we don't have many.